PUSH_TO_FILE.md

```diff
@@ -0,0 +1,302 @@
+
+Below is a list of Annotations that are used for basic Secrets Provider configurationv 
+and to write the secrets to file.
+All annotations begin with `conjur.org/` so they remain unique.
+Push to File Annotations are organized by "secret groups". A secrets group is a logical grouping of application secrets, typically belonging to a particular component of an application deployment (e.g. all secrets related to a backend database). Each group of secrets is associated with a specific destination file.
+
+Please refer to the
+[Secrets Provider documentation](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Integrations/k8s-ocp/cjr-k8s-secrets-provider-ic.htm)
+for a description of each environment variable setting:
+
+| K8s Annotation  | Equivalent<br>Environment Variable | Description, Notes |
+|-----------------------------------------|---------------------|----------------------------------|
+| `conjur.org/authn-identity`         | `CONJUR_AUTHN_LOGIN`  | Required value. Example: `host/conjur/authn-k8s/cluster/apps/inventory-api` |
+| `conjur.org/container-mode`         | `CONTAINER_MODE`      | Allowed values: <ul><li>`init`</li><li>`application`</li></ul>Defaults to `init`.<br>Must be set to init for Push to File mode.|
+| `conjur.org/secrets-destination`    | `SECRETS_DESTINATION` | Allowed values: <ul><li>`file`</li><li>`k8s_secret`</li></ul> |
+| `conjur.org/k8s-secrets`            | `K8S_SECRETS`         | This list is ignored when `conjur.org/secrets-destination` annotation is set to **`file`** |
+| `conjur.org/retry-count-limit`      | `RETRY_COUNT_LIMIT`   | Defaults to 5
+| `conjur.org/retry-interval-sec`     | `RETRY_INTERVAL_SEC`  | Defaults to 1 (sec)              |
+| `conjur.org/debug-logging`          | `DEBUG`               | Defaults to `false`              |
+| conjur.org/conjur-secrets.{secret-group}      | Push to File config is not available with environmental variables | List of secrets to be retrieved from Conjur. Each entry can be either:<ul><li>A Conjur variable path</li><li> A key/value pairs of the form `<alias>:<Conjur variable path>` where the `alias` represents the name of the secret to be written to the secrets file |
+| conjur.org/secret-file-path.{secret-group}    | Push to File config is not available with environmental variables |  Relative path for secrets file or directory to be written. This path is assumed to be relative to the respective mount path for the shared secrets volume for each container (for the Secrets Provider container, the mount path is `/conjur/secrets/`). Since this is a relative path, values beginning with `/` are rejected and cause the Secrets Provider to abort. The file path must also include a file name (i.e. must not be a directory). Values ending with `/` are rejected and cause the Secrets Provider to abort.
+| conjur.org/secret-file-format.{secret-group}  | Push to File config is not available with environmental variables | Allowed values:yaml (default) |
```

Comment:
Annotations aren't wrapped in backtics.
```suggestion
| `conjur.org/conjur-secrets.{secret-group}`      | Push to File config is not available with environmental variables | List of secrets to be retrieved from Conjur. Each entry can be either:<ul><li>A Conjur variable path</li><li> A key/value pairs of the form `<alias>:<Conjur variable path>` where the `alias` represents the name of the secret to be written to the secrets file |
| `conjur.org/secret-file-path.{secret-group}`    | Push to File config is not available with environmental variables |  Relative path for secrets file or directory to be written. This path is assumed to be relative to the respective mount path for the shared secrets volume for each container (for the Secrets Provider container, the mount path is `/conjur/secrets/`). Since this is a relative path, values beginning with `/` are rejected and cause the Secrets Provider to abort. The file path must also include a file name (i.e. must not be a directory). Values ending with `/` are rejected and cause the Secrets Provider to abort.
| `conjur.org/secret-file-format.{secret-group}`  | Push to File config is not available with environmental variables | Allowed values:yaml (default) |
```